src/main/java/frc/robot/subsystems/shooter/ShooterSS.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.components.TrigonTalonFX;
+import frc.robot.constants.RobotConstants.ShooterConstants;
+import frc.robot.subsystems.TestableSubsystem;
+import frc.robot.utilities.EncoderConversions;
+
+public class ShooterSS extends SubsystemBase implements TestableSubsystem {
+    private final TrigonTalonFX masterMotor;
+
+    public ShooterSS() {
+        masterMotor = ShooterConstants.LEFT_MOTOR;
+        ShooterConstants.RIGHT_MOTOR.follow(masterMotor);
+        ShooterConstants.LEFT_MOTOR.follow(masterMotor);
+    }
+
+    /**
+     * @param velocityRPM to be set to the motors
+     */
+    public void setVelocityRPM(double velocityRPM) {
+        masterMotor.set(ControlMode.Velocity, EncoderConversions.RPMToFalcon(velocityRPM));
```

Comment:
after you fix everything else update from main and make this work with the current PID code. like in loader

Reply:
That's now